components/NewsModal/index.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Box, Button, Dialog, Grid, IconButton, Slide, Typography, useMediaQuery, useTheme } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import Image from 'next/image';
+import Link from 'next/link';
+import useRouter from 'hooks/useRouter';
+import { useTranslation } from 'react-i18next';
+
+const NEWS_READ_KEY = 'news_read';
+
+const getReadNews = () => {
+  const storedNews = localStorage.getItem(NEWS_READ_KEY);
+  return storedNews ? JSON.parse(storedNews) : [];
+};
+
+const isNewsRead = (id: string) => {
+  const readNews = getReadNews();
+  return readNews.includes(id);
+};
+
+const NewsModal = () => {
+  const { t } = useTranslation();
+  const news = useMemo(
+    () => [
+      {
+        id: 'Security Hub',
+        title: t('Security Hub'),
+        description: [
+          t('Stay up to date with all the security measures we put in place to keep the Protocol safe.'),
+          t("From smart contract audits to revoking token allowances, you'll find all the details right here."),
```

Comment:
why did this end up with double quotes? weird prettier didn't fix that 🤔

Reply:
there is a single quote in `you'll`

Reply:
ohh, ok!